amr-wind/equation_systems/tke/source_terms/KsgsM84Src.H

```diff
@@ -0,0 +1,41 @@
+#ifndef KSGSM84SRC_H
+#define KSGSM84SRC_H
+
+#include "amr-wind/equation_systems/tke/TKESource.H"
+
+namespace amr_wind {
+namespace pde {
+namespace tke {
+
+class KsgsM84Src : public TKESource::Register<KsgsM84Src>
```

Comment:
```suggestion
/** One-equation Ksgs LES turbulence model
 *   \ingroup turb_model
 *
 *  Reference to Moeng paper
 */
class KsgsM84Src : public TKESource::Register<KsgsM84Src>
```

so that it gets picked up in [Turbulence models docs](https://exawind.github.io/amr-wind/group__turb__model.html)